### ISSUE TITLE
cannot find reaper

### ISSUE THREAD

**Issue by Snevzor** [Cybolic/LinReaper#4]
Installed under (the Exe is in this folder and works): /home/sven/reaper/
./reaper.sh output: wine: cannot find 'C:\Program Files\REAPER\Reaper.exe'

It worked before so it must be broken because of a Wine update or something?

**Comment by Snevzor** [comment]
Script 'searches' for the folder 'REAPER' in Program Files, but it was installed to Program Files (x86). Copied the folder REAPER to Program Files (x86) and the script works again.
The script does not evaluate if the folder REAPER exists, but it does seem to check if 'Program Files' exists?

**Comment by Cybolic** [comment]
That is correct. The run script only checks the registry to see where Reaper would have been installed and assumes it has been installed there. I should add proper checks.

**Comment by Snevzor** [comment]
Okay! Well, if I could I would do a code suggestion but I have zero experience in shellscripting.
